packages/shared/catalogs/sagas.ts

```diff
@@ -100,7 +105,10 @@ function* fetchWearablesFromCatalyst(filters: WearablesRequestFilters) {
       // Fetch published collections
       const urnCollections = collectionIds.filter((collectionId) => collectionId.startsWith('urn'))
       if (urnCollections.length > 0) {
-        const zoneWearables: PartialWearableV2[] = yield client.fetchWearables({ collectionIds: urnCollections })
+        const zoneWearables: PartialWearableV2[] = yield jsonFetch(
```

Comment:
```suggestion
        const zoneWearables: PartialWearableV2[] = yield call(jsonFetch,
```

What would happen in case of failure in the response?